### DISCUSSION TITLE
Missing imaging data in Release 5.0?

### DISCUSSION THREAD

**Discussion by maloneytc** [now-i-know-my-abcd/docs#57] [Stats & Code Help] [answered]
Hello,

When I query the number of participants with imaging data in the miNDAR's I've created for release 4 and release 5, I find that there are thousands of fewer participants in release 5 than there were in release 4.  For example, the number of participants with a minimally processed baseline T1 scan went from 11,402 in release 4 to 10,686 in release 5. The number of SST scans went from 18,952 in release 4 to 14,905 in release 5.

I do not see anything in the release documentation about this and was wondering if anyone else knows why the numbers went down?  Thanks!

T1 Query:
SELECT count(*) FROM FMRIRESULTS01 WHERE DERIVED_FILES LIKE '%baselineYear1Arm1_ABCD-MPROC-T1%';

**Comment by maloneytc** [comment]
Comparing the number of patients with at least one 2-year follow-up SST fMRI scan, I found 286 participants in release 5 that are not in release 4 and 1,320 participants in release 4 that are not in release 5.

Is anyone else seeing a reduction in the number of participants with imaging data in release 5? Anyone have insight as to why?

**Reply by mangstad** [reply, marked as answer]
Not sure if you've seen this, but missing mproc data is now reported as a known issue in release 5.1 but is not expected to be fixed until release 6.0.
https://wiki.abcdstudy.org/release-notes/start-page.html
